src/OrchardCore.Modules/OrchardCore.Lists/RemotePublishing/MetaWeblogHandler.cs

```diff
@@ -165,9 +166,9 @@ public void SetCapabilities(XElement options)
 
             string directoryName = Path.GetDirectoryName(name);            
             string filePath = _mediaFileStore.Combine(directoryName, Path.GetFileName(name));
-            bool saved = await _mediaFileStore.TrySaveStreamAsync(filePath, new MemoryStream(bits));
+            await _mediaFileStore.CreateFileFromStream(filePath, new MemoryStream(bits));
```

Comment:
Should the stream be in a using?

Reply:
Well I didn't change this, but anyway I think in this case it's not necessary since it's a memory stream only. Both the stream and the underlying buffer `bits` will be GC'ed when they go out of scope, and a memory stream holds no open file handles or anything like that.